Modules/CalcSections-3_0.lua

```diff
@@ -823,11 +823,26 @@ return {
 } }
 } },
 { 1, "EleAilments", 1, colorCodes.OFFENCE, {{ defaultCollapsed = false, label = "Elemental Ailments", data = {
+	{ label = "Inc. Scorch Effect", haveOutput = "ScorchEffectMod", flag = "scorch", { format = "{0:output:ScorchEffectMod}%",
+		{ label = "Player modifiers", modName = "EnemyScorchEffect", cfg = "skill" },
+		{ label = "Enemy modifiers", modName = "SelfScorchEffect", enemy = true },
+	}, },
+	{ label = "Chance to Scorch", haveOutput = "ScorchChance", flag = "scorch", { format = "{0:output:ScorchChance}%",
+		{ breakdown = "MainHand.ScorchChance" }, 
+		{ breakdown = "OffHand.ScorchChance" }, 
+		{ breakdown = "ScorchChance" }, 
+		{ label = "Player modifiers", modName = "EnemyScorchChance", cfg = "skill" }, 
+		{ label = "Enemy modifiers", modName = "SelfScorchChance", enemy = true },
+	}, },
+	{ label = "Scorch Dur. Mod", haveOutput = "ScorchDurationMod", flag = "scorch", { format = "x {2:output:ScorchDurationMod}",
+		{ label = "Player modifiers", modName = "EnemyScorchDuration", cfg = "skill" },
+		{ label = "Enemy modifiers", modName = "SelfScorchEffect", enemy = true },
+	}, },
 	{ label = "Inc. Chill Effect", flag = "chill", { format = "{0:output:ChillEffectMod}%",
 		{ label = "Player modifiers", modName = "EnemyChillEffect", cfg = "skill" },
 		{ label = "Enemy modifiers", modName = "SelfChillEffect", enemy = true },
 	}, },
-	{ label = "Chill Dur. Mod:", flag = "chill", { format = "x {2:output:ChillDurationMod}",
```

Comment:
For completely unrelated changes like this one, please refactor into another PR.